editor/editor_node.cpp

```diff
@@ -6102,6 +6141,13 @@ EditorNode::EditorNode() {
 	FileAccess::set_file_close_fail_notify_callback(_file_access_close_error_notify);
 
 	waiting_for_first_scan = true;
+
+	_dimming = false;
+	_dim_time = 0.0f;
+	_dim_timer = memnew(Timer);
+	_dim_timer->set_wait_time(0.01666f);
```

Comment:
If `"interface/dim_transition_time"` is more than `0.1666f`, would this cause a premature stop of the timer?